homeassistant/components/sensor/plex.py

```diff
@@ -102,7 +102,9 @@ def state(self):
     @property
     def unit_of_measurement(self):
         """Return the unit this state is expressed in."""
-        return "Watching"
+        if self._state > 1:
```

Comment:
Not allowed. unit needs to be static.

Reply:
Okay, I will change it, but what is the reason?